paimon-vfs/pom.xml

```diff
@@ -49,43 +49,43 @@ under the License.
             <groupId>org.apache.paimon</groupId>
             <artifactId>paimon-core</artifactId>
             <version>${project.version}</version>
-            <scope>test</scope>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.paimon</groupId>
             <artifactId>paimon-core</artifactId>
             <classifier>tests</classifier>
             <version>${project.version}</version>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
Why change to provided?

Reply:
It's a misoperation, I will revert this.